controllers/api.controller.js

```diff
@@ -39,4 +40,23 @@ function getArticles(req, res, next) {
     .catch(next)
 }
 
-module.exports = { getApi, getTopics, getArticleByID, getArticles }
+function getArticleComments(req, res, next) {
+  const { article_id } = req.params
+
+  fetchArticleComments(article_id)
+    .then((comments) => {
+      if (!comments.length) {
```

Comment:
Remember that .then() is for handling a promises resolving - i.e. we got the data everything is successful - and .catch is to handle errors - theres a problem and the promise has rejected - therefore we should make sure the model resolves or rejects so the controller can send the appropriate response in the appropriate place!